### PR TITLE
Bug 1569658 clarify purpose of enabling efk audit container engine

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1569658

### CODE REVIEW

**Comment by vikram-redhat** [comment]
I have requested @anpingli review this PR because we haven't had a response from Eric Jones.

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -376,14 +376,23 @@ starting up, using this may cause a delay in ES receiving current log records.
 
 |`openshift_logging_fluentd_audit_container_engine`
 |When `openshift_logging_fluentd_audit_container_engine` is set to `true`, the
-audit log of the container engine is collected and stored in ES.
+audit log of the container engine is collected and stored in ES. Enabling this
+variable will allow the EFK watch the specified audit log file or the
```
Comment (mburke5678) [inline install_config/aggregate_logging.adoc]: s/will allow the EFK watch/allows the EFK to watch

File: install_config/aggregate_logging.adoc [hunk 2]
```diff
@@ -376,14 +376,23 @@ starting up, using this may cause a delay in ES receiving current log records.
...
+default `/var/log/audit.log` file, and collect audit information for the container
+engine for the platform, then put it into Kibana.
```
Comment (mburke5678) [inline install_config/aggregate_logging.adoc]: s/and collect/collect ??

File: install_config/aggregate_logging.adoc [hunk 3]
```diff
@@ -376,14 +376,23 @@ starting up, using this may cause a delay in ES receiving current log records.
...
 
 |`openshift_logging_fluentd_audit_file`
-|Location of audit log file. The default is `/var/log/audit/audit.log`.
+|Location of audit log file. The default is `/var/log/audit/audit.log`. Enabling this
+variable will allow the EFK watch the specified audit log file or the
+default `/var/log/audit.log` file, and collect audit information for the container
+engine for the platform, then put it into Kibana.
```
Comment (mburke5678) [inline install_config/aggregate_logging.adoc]: Same as above

File: install_config/aggregate_logging.adoc [hunk 4]
```diff
@@ -376,14 +376,23 @@ starting up, using this may cause a delay in ES receiving current log records.
...
 
 |`openshift_logging_fluentd_audit_pos_file`
 |Location of the Fluentd `in_tail` position file for the audit log file. The default is
-`/var/log/audit/audit.log.pos`.
+`/var/log/audit/audit.log.pos`. Enabling this
+variable will allow the EFK watch the specified audit log file or the
+default `/var/log/audit.log` file, and collect audit information for the container
+engine for the platform, then put it into Kibana.
```
Comment (mburke5678) [inline install_config/aggregate_logging.adoc]: Same as above

**Comment by mburke5678** [comment]
@bmcelvee Couple of nits. Otherwise LGTM

**Comment by bmcelvee** [comment]
Thanks, @mburke5678! :cheese:

**Pull request merged**